cleantalk/antispam/acp/main_module.php

```diff
@@ -10,6 +10,8 @@
 
 namespace cleantalk\antispam\acp;
 
+use cleantalk\antispam\library\Cleantalk\Common\API;
```

Comment:
Why this commit is here?
It doesn't relate to statistics in any way. It  should be in some other branch.

Reply:
Yes, but so it happened